docs/_docs/user-guide/eldritch.md

```diff
@@ -223,10 +228,12 @@ The <b>pivot.arp_scan</b> method is being proposed to allow users to enumerate h
 The <b>pivot.port_forward</b> method is being proposed to providde socat like functionality by forwarding traffic from a port on a local machine to a port on a different machine allowing traffic to be relayed.
 
 ### pivot.ncat
-`pivot.ncat(address: str, port: int, data: str, str: portocol ) -> str`
+`pivot.ncat(address: str, port: int, data: str, portocol: str ) -> str`
```

Comment:
portocol, or protocol?

Reply:
Or is this intentional :) 

https://github.com/KCarretto/realm/blob/995d1beca806d43465e58ca1767c4b7a950b538b/implants/eldritch/src/pivot.rs#LL83C80-L83C80

https://github.com/KCarretto/realm/blob/995d1beca806d43465e58ca1767c4b7a950b538b/implants/eldritch/src/pivot.rs#L78

https://github.com/KCarretto/realm/blob/995d1beca806d43465e58ca1767c4b7a950b538b/implants/eldritch/src/pivot/ncat_impl.rs#L57

Reply:
😅 just wish I knew how to spell.